flowzone.yml

```
@@ -1407,7 +1407,7 @@ jobs:
             }
 
             // Check if PR is from Renovate
-            const isRenovate = pr.user.type === 'Bot' && pr.user.login.includes('renovate');
+            const isRenovate = (pr.labels || []).some(label => ((typeof label === 'string' ? label : label.name) || '').toLowerCase() === 'renovate');
```

This can be somewhat simplified by using optional chaining:
```suggestion
            const isRenovate = pr.labels?.some(label => (typeof label === 'string' ? label : label.name)?.toLowerCase() === 'renovate') === true;
```